File: Completed/js/almostIncreasingSequence/almostIncreasingSequence.py
def almostIncreasingSequence(a):
    flag = False
    for i in range(len(a) - 1):
        if(flag):
            return False
        else if(a[i] >= a[i+1]):
            flag = True
    return True


print(almostIncreasingSequence([1, 3, 2, 1]))
print(almostIncreasingSequence([1, 3, 2]))

File: Completed/js/almostIncreasingSequence/almostIncreasingSequence.js
function almostIncreasingSequence(a) {
    let flag = false;
    for (let i = 0; i < a.length - 1; i++) {
        if (flag) {
            return false;
        } else if (a[i] >= a[i + 1]) {
            flag = true;
        }
    }
    return true
}

console.log(almostIncreasingSequence([1, 3, 1, 2]))
console.log(almostIncreasingSequence([1, 3, 2]))